### PR TITLE
Run master/workers on AWS

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: locust-runner.py [hunk 1]
```diff
@@ -1,34 +0,0 @@
-#!/usr/bin/env python
```
Comment (mstriemer) [inline locust-runner.py:1]: Was this not used at all?

Comment (mstriemer) [inline locust-runner.py:1]: I see it's just using `locust` now.

**Comment by mstriemer** [comment]
This seems good to me. Would be nice to have some way to automate this. All I can think of is fabric but I'm sure there's a new hotness.

**Comment by kumar303** [comment]
Yeah, I think the next step is to automate this with boto and fabric (or something)

File: locust-runner.py (hunk 1 above)
Comment (kumar303) [inline locust-runner.py:1]: This script was never needed. I wrote it when I couldn't get env vars working in the yaml file but I upgraded docker-compose and switched some things (maybe it needed double quotes?); now it works.

**Pull request merged**